src/index.html

```diff
@@ -5,8 +5,405 @@
+            <a class="header__nav-link" href="#Smartphones">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              class="header__nav-link"
+              href="#Laptops &amp; computers"
+              data-qa="nav-hover"
+            >
+              Laptops &amp; computers
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a class="header__nav-link" href="#Gadgets">
+              Gadgets
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a class="header__nav-link" href="#Tablets">
+              Tablets
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a class="header__nav-link" href="#Photo">
+              Photo
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a class="header__nav-link" href="#Video">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+
+      <div class="product" data-qa="card">
+        <div class="product__card">
+          <img
+            type="image"
+            class="product__image"
+            src="./images/imac.jpeg"
+            alt="image"
+          >
+        </div>
+        <div class="product__name">
+          <section class="product__name-apple">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </section>
+          <section class="product__name-code">
+            Product code: 195434
+          </section>
+        </div>
+        <div class="product__ratings">
+          <div class="product__stars product__stars--4">
+            <div class="product__stars-star"></div>
+            <div class="product__stars-star"></div>
+            <div class="product__stars-star"></div>
+            <div class="product__stars-star"></div>
+            <div class="product__stars-star"></div>
+          </div>
+          <div class="product__ratings-reviews">Reviews: 5</div>
+        </div>
+        <div class="product__price">
+          <section class="product__price-price">Price:</section>
+          <section>$2,199</section>
+        </div>
+        <div class="product__buy">
+          <a
+            href="#"
+            class="product__buy-link"
+            data-qa="card-hover"
+          >
+            Buy
+          </a>
+        </div>
+      </div>
+
+      <div class="product">
+        <div class="product__card">
+          <img
+            type="image"
+            class="product__image"
+            src="./images/imac.jpeg"
+            alt="image"
+          >
+        </div>
+        <div class="product__name">
+          <section class="product__name-apple">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </section>
+          <section class="product__name-code">
+            Product code: 195434
+          </section>
```

Comment:
Use semantic tags for text content like `<p>`, `<h2>`, `<span>` and so on

Fix everywhere